index.html

```diff
@@ -2825,125 +2827,265 @@ <h3>Working with personal names</h3>
+
 	<div class="req" id="loc_names_nee">
 	<p class="advisement">It may be better for a form to ask for 'Previous name' rather than 'Maiden name' or 'née'.</p>
-	<details class="links"><summary>explanations &amp; examples</summary>
+	<details class="links"><summary>source</summary>
 	<p><a href="https://www.w3.org/International/questions/qa-personal-names#otherthings">Other things</a>, in <cite>Personal Names Around the World</cite>.</p>
 	</details>
+	<details class="links"><summary>examples</summary>
+	<p><a href="https://www.w3.org/International/questions/qa-personal-names#examples">Examples of differences</a> in <cite>Personal Names Around the World</cite>.</p>
+	</details>
 	</div>
 
+<p>You should also not simply assume that name adoption goes from husband to wife. Sometimes men take their wife's name on marriage. It may be better, in these cases, for a form to say 'Previous name' than 'Maiden name' or 'née'.</p>
+
 
 	<div class="req" id="loc_names_latin_native">
-	<p class="advisement">You may want to store the name in both Latin and native scripts, in which case you probably need to ask the user to submit their name in both native script and Latin-only form, as separate items.</p>
-	<details class="links"><summary>explanations &amp; examples</summary>
+	<p class="advisement">You probably need to store the name in both Latin and native scripts, in which case you will need to ask the user to submit their name in both native script and Latin-only form, as separate items.</p>
+	<details class="links"><summary>source</summary>
+	<p><a href="https://www.w3.org/International/questions/qa-personal-names#encoding">Implications for character support</a>, in <cite>Personal Names Around the World</cite>.</p>
+	</details>
+	<details class="links"><summary>examples</summary>
+	<p><a href="https://www.w3.org/International/questions/qa-personal-names#examples">Examples of differences</a> in <cite>Personal Names Around the World</cite>.</p>
+	</details>
+	</div>
+
+<p>The need for multiple fields will depend to some extent on what you are collecting people’s names for, and how you intend to use them.</p>
+<ul>
+<li>Are you collecting the person’s name just to have an identifier in your system?  If so, it may not matter whether the name is stored in ASCII-only or native script.</li>
+
+<li>Or will they be called by name on a welcome page or in correspondence?  If you will correspond using their name on pages written in their language, it would seem sensible to have the name in the native script.</li>
+
+<li>Is it important for people in the organization that handles queries to be able to recognise and use the person’s name?  If so, you may need a Latin transcription.</li>
```

Comment:
Might be useful to make the spelling more consistent (for 'organization' and 'recognise').